gcloud/_apitools/buffered_stream.py

```diff
@@ -13,32 +13,32 @@ class BufferedStream(object):
     """Buffers a stream, reading ahead to determine if we're at the end."""
 
     def __init__(self, stream, start, size):
-        self.__stream = stream
-        self.__start_pos = start
-        self.__buffer_pos = 0
-        self.__buffered_data = self.__stream.read(size)
-        self.__stream_at_end = len(self.__buffered_data) < size
-        self.__end_pos = self.__start_pos + len(self.__buffered_data)
-
-    def __str__(self):
+        self._stream = stream
+        self._start_pos = start
+        self._buffer_pos = 0
+        self._buffered_data = self._stream.read(size)
+        self._stream_at_end = len(self._buffered_data) < size
+        self._end_pos = self._start_pos + len(self._buffered_data)
+
+    def __str__(self):  # pragma: NO COVER
         return ('Buffered stream %s from position %s-%s with %s '
-                'bytes remaining' % (self.__stream, self.__start_pos,
-                                     self.__end_pos, self._bytes_remaining))
+                'bytes remaining' % (self._stream, self._start_pos,
+                                     self._end_pos, self._bytes_remaining))
 
     def __len__(self):
-        return len(self.__buffered_data)
+        return len(self._buffered_data)
 
     @property
     def stream_exhausted(self):
-        return self.__stream_at_end
+        return self._stream_at_end
 
     @property
     def stream_end_position(self):
-        return self.__end_pos
+        return self._end_pos
 
     @property
     def _bytes_remaining(self):
-        return len(self.__buffered_data) - self.__buffer_pos
+        return len(self._buffered_data) - self._buffer_pos
 
     def read(self, size=None):  # pylint: disable=invalid-name
```

Comment:
Dunno -- it is a carryover from the old codebase.  We can review when we enable `pylint` post-cleanups.